{"name": "test_negative_calorie_count", "status": "skipped", "statusDetails": {"message": "XFAIL\n\nAssertionError: If dietry-fiber is greater than carbs then calorie count comes negative which is not as expected\nassert -15 == 265.0", "trace": "self = <test_cases.test_calorie_count.Test_001 object at 0x000001716B2C0610>\n\n    @pytest.mark.xfail\n    def test_negative_calorie_count(self):\n        file = self.test_calorie_sheet()\n        df = pd.read_excel(file)\n        print(df)\n        print(df.iloc[2].tolist())  # Get the third Row from excel using Pandas as a List\n        protein, carbs, dietry_fiber, fat, alcohol, expected = df.iloc[2].tolist()\n        total_calories = Test_001.calculate_calories(protein, carbs, dietry_fiber, fat, alcohol)\n        print(f'total_calories is {total_calories}')\n>       assert total_calories == expected, \"If dietry-fiber is greater than carbs then calorie count comes negative which is not as expected\"\nE       AssertionError: If dietry-fiber is greater than carbs then calorie count comes negative which is not as expected\nE       assert -15 == 265.0\n\ntest_calorie_count.py:64: AssertionError"}, "start": 1695593953572, "stop": 1695593953593, "uuid": "8d6b38d2-2698-40e4-9ceb-9a044705e89b", "historyId": "41f6076f4e077c6a1d7d00b65c40ccfc", "testCaseId": "41f6076f4e077c6a1d7d00b65c40ccfc", "fullName": "test_calorie_count.Test_001#test_negative_calorie_count", "labels": [{"name": "tag", "value": "@pytest.mark.xfail"}, {"name": "suite", "value": "test_calorie_count"}, {"name": "subSuite", "value": "Test_001"}, {"name": "host", "value": "Priya-Lenovo"}, {"name": "thread", "value": "13632-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calorie_count"}]}